packages/terra-date-input/tests/wdio/date-input-spec.js

```diff
@@ -396,7 +417,22 @@ Terra.describeViewports('Date Input', ['medium'], () => {
       Terra.it.matchesScreenshot();
     });
 
-    describe('Year Input DOWN_ARROW cycles to 9999 when the year has reached 1', () => {
+    describe('Year Input DOWN_ARROW cycles to 2100 when the year has reached 1900', () => {
+      before(() => {
+        browser.url('/#/raw/tests/terra-date-input/date-input/default-date-input');
+        browser.refresh();
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
+        Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
+
+        browser.click('input[name="terra-date-year-date-input"]');
+        browser.keys(['1900', 'ArrowDown']);
+      });
+
+      Terra.it.matchesScreenshot();
+    });
+
```

Comment:
The Terra.it helpers are being deprecated. Can we update?
```suggestion
    it('Year Input DOWN_ARROW cycles to 2100 when the year has reached 1900', () => {
      browser.url('/#/raw/tests/terra-date-input/date-input/default-date-input');
      browser.refresh();
        
      // Removes the blinking cursor to prevent screenshot mismatches.
      Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
      Terra.hideInputCaret('input[name="terra-date-year-date-input"]');

      browser.click('input[name="terra-date-year-date-input"]');
      browser.keys(['1900', 'ArrowDown']);

      Terra.validates.screenshot();
    });

```